### PR TITLE
Get user's location using geolocation api

### LINKED ISSUE #3: Using Geolocation API to get the users location
Using the Geolocation API to get the users location as the first weather results displayed, instead of just New York City.

Read more [here](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API).

### PR DESCRIPTION
**Pull request by siddhi-244**
Resolves #3 
Called the geolocation api in `App.js` and fetched co-ordinates of the user. 

## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -8,23 +8,44 @@ function App() {
   const [city, setCity] = useState("New York City")
   const [results, setResults] = useState(null);
 
+  function success(position) {
+    var latitude = position.coords.latitude, longitude = position.coords.longitude;
```
Comment (Devansu-Yadav) [inline src/App.js]: Better to make use of `const` instead of `var` as a good practice in JS.

File: src/App.js [hunk 2]
```diff
@@ -8,23 +8,44 @@ function App() {
   const [city, setCity] = useState("New York City")
   const [results, setResults] = useState(null);
 
+  function success(position) {
```
Comment (Devansu-Yadav) [inline src/App.js]: I'd suggest sticking to using Arrow functions for consistency in code.

Comment (Devansu-Yadav) [inline src/App.js]: The function should be named more appropriately, something like `fetchUserLocation`.

**Review by Devansu-Yadav — CHANGES_REQUESTED** [review]
I mentioned some cosmetic changes, and the rest looks fine. Also, the deploy preview is breaking I guess.

**Comment by siddhi-244** [comment]
Will work on these. The project is running fine in the local machine ! @Devansu-Yadav. Let me see what's the issue !

**Comment by siddhi-244** [comment]
@Devansu-Yadav Done the changes!

**Review by Devansu-Yadav — APPROVED** [review]
LGTM 👍

**Comment by akshitadixit** [comment]
image
@Devansu-Yadav remember to resolve conversations once you're satisfied with that particular change

**Pull request merged**